docs/disaster-recovery.md

```diff
@@ -50,26 +50,35 @@ cf delete-service <instance-name>
 Then recreate the lost postgres database instance using the following make recipes `deploy-plan` and `deploy`.  To see the proposed changes:
 
 ```
-TAG=$(cf app teacher-training-api-prod | grep -Po "docker image:\s+\S+:\K\w+")
-make <env> deploy-plan passcode=<my-passcode> IMAGE_TAG=${TAG}
+TAG=$(cf app teacher-training-api-qa | awk -F : '$1 == "docker image" {print $3}')
+make <env> deploy-plan PASSCODE=<my-passcode> IMAGE_TAG=${TAG}
 ```
 To apply proposed changes i.e. create new database instance:
 ```
-TAG=$(cf app teacher-training-api-prod | grep -Po "docker image:\s+\S+:\K\w+")
-make <env> deploy-plan passcode=<my-passcode> CONFIRM_RESTORE=YES IMAGE_TAG=${TAG}
+TAG=$(cf app teacher-training-api-qa | awk -F : '$1 == "docker image" {print $3}')
+make <env> deploy PASSCODE=<my-passcode> IMAGE_TAG=${TAG}
 ```
 This will create a new postgres database instance as described in the terraform configuration file.
 
 \* ~20 minutes to create a new database instance and 2-5 min restore time of sanitised db and testing in QA
 
 ### Restore Data From Nightly Backup
 
-Once the lost database instance has been recreated, the last nightly backup will need to be restored. To achieve this, use the following makefile recipe: `restore-data-from-nightly-backup`. The following will need to be set: `passcode` (a [GOV.UK PaaS one-time passcode](https://login.london.cloud.service.gov.uk/passcode)), `CONFIRM_PRODUCTION=YES`,  `CONFIRM_RESTORE=YES` and `BACKUP_DATE="yyyy-mm-dd"`.  You will need to be logged into GovUK PaaS and Azure using the `az` and `cf` CLIs.
+Once the lost database instance has been recreated, the last nightly backup will need to be restored. To achieve this, use the following makefile recipe: `restore-data-from-nightly-backup`. The following will need to be set: `passcode` (a [GOV.UK PaaS one-time passcode](https://login.london.cloud.service.gov.uk/passcode)), `CONFIRM_PRODUCTION=YES`,  `CONFIRM_RESTORE=YES` and `BACKUP_DATE="yyyy-mm-dd"`.
+
+You will need to be logged into GovUK PaaS and Azure using the `az` and `cf` CLIs.  You will need to raise a [PIM](https://docs.microsoft.com/en-us/azure/active-directory/privileged-identity-management/pim-resource-roles-activate-your-roles) request to elevate your credentials for a production restore.  A collegue will need to approve this for you.
 
 ```
-make production restore-data-from-nightly-backup CONFIRM_PRODUCTION=YES CONFIRM_RESTORE=YES BACKUP_DATE="yyyy-mm-dd"
+# env is the target environment in the make file e.g. 'production'
+# space is the name of the environment in GOV.UK PaaS, eg 'bat-prod'
+az login
+cf login -o dfe -s <space> -u my.name@digital.education.gov.uk
+make <env> restore-data-from-nightly-backup BACKUP_DATE="yyyy-mm-dd" CONFIRM_RESTORE=YES CONFIRM_PRODUCTION=YES
 ```
 
+##TO DO: change permissions on files chmod +x
```

Comment:
It be good practice to list notable permissions required.

FYI
https://stackoverflow.com/questions/23667859/why-is-git-clone-changing-file-permissions

Reply:
I've replaced that TO DO with something more useful :)

It looks like the permissions issue might be WSL related, we're still investigating.  File permissions are fixed now though